### DISCUSSION TITLE
Installation on LinuxLite failing

### DISCUSSION THREAD

**Discussion by TekyTech** [vicwomg/pikaraoke#425] [Q&A] [answered]
I've installed an older version a while ago and it went smooth, the installer did pretty much everything. Dealing with Python, FFmpeg and Chromium (Chrome) Driver was easy, if required at all.

The new version installation fails while seemingly following the same steps. 

**Could anybody please point to what am i doing wrong?**

1. **Install git** (LinuxLite does not come with git preinstalled)
2. **git clone** ... pikaraoke (I was instructed to use git rather than a local install to be able to install updates smoothly in the future also using git)
3. Install **python venv**
4. **sudo ./setup.sh** 
5. in the pikaraoke/scripts run ./pikaraoke.sh

It does not start with the error:
can't open file '/home/user/pikaraoke/scripts/app.py': [Errno 2] No such file or directory

Although there is app.py in a different folder: pikaraoke/pikaraoke (not sure why developers chose to use another pikaraoke folder inside a pikaraoke folder). I thought I would edit the pikaraoke.sh file to refer to the app.py location explicitly: python3 /home/user/pikaraoke/pikaraoke/app.py $@

Ok, edited that, it starts but more errors:
File "/home/user/pikaraoke/pikaraoke/app.py", line 37, in <module>
    from pikaraoke import VERSION, karaoke
ModuleNotFoundError: No module named 'pikaraoke'

It just keeps going on...

Is there any script that works or can this one be fixed?

Should i post this in the BUGS report rather than Discussions and FAQ?

Thank you.

**Comment by vicwomg** [comment]
Pikaraoke is installed via pip now. There is no need to run any scripts. Please see the README for the latest install instructions

**Reply by TekyTech** [reply]
May I ask you please, what’s the best way to upgrade to a newer version since the installation changed to pip?

I mean if the older version was installed like this a while ago:
pip install pikaraoke

P.S.  Wow, you’ve done a lot of work over the holidays. Very productive! And thank you.

**Reply by vicwomg** [reply]
This is the standard way to update python packages from pip: `pip install pikaraoke --upgrade`

**Comment by vicwomg** [comment, marked as answer]
Really, to summarize, just run:
`pip install pikaraoke`

Then launch with `pikaraoke` (it is installed in the PATH)

**Comment by TekyTech** [comment]
Thank you so much, as always! Works beautifully!!!

I saw the Readme but I thought the instructions you gave me last time were more accurate than the Readme, which I thought might have been outdated. Yes, everything according to Readme worked like a charm. I reinstalled the system on an older laptop used for travel. 

Family is happy again :-)
